src/containers/DAO/pages/DAOCreate/DAOCreate.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { MarkDownEditor, MarkDownViewer } from 'components';
+
+export const DAOCreate: React.FC = () => {
+	return (
+		<>
+			<h2>Markdown Editor Example</h2>
+			<MarkDownEditor text="" />
+
+			<hr />
+
+			<h2>Markdown Viewer Example</h2>
+			<MarkDownViewer text="Hello World" />
```

Comment:
I know it is just example, but can we use longer and more complex markdown text as an example? 💪